apps/app/components/LectureForm/index.js

```diff
@@ -0,0 +1,191 @@
+import {
+  getNinjaEvents,
+  listEvents,
+  listGuardians,
+  listMentors,
+} from "bokkenjs";
+import * as api from "bokkenjs";
+
+const { Title } = Typography;
+
+export default function LectureForm({ id }) {
+  const router = useRouter();
+  const [form] = Form.useForm();
+  const [selectedMentor, setSelectedMentor] = useState({});
+  const [selectedNinja, setSelectedNinja] = useState({});
+  const [selectedEvent, setSelectedEvent] = useState({});
+
+  const [mentors, setMentors] = useState([]);
+  useEffect(() => {
+    listMentors().then((response) => {
+      setMentors(response.data);
+    });
+  }, []);
+
+  const [events, setEvents] = useState([]);
+  useEffect(() => {
+    listEvents().then((response) => {
+      setEvents(response.data);
+    });
+  }, []);
+
+  const [ninjas, setNinjas] = useState([]);
+  useEffect(() => {
+    events.map((event) => {
+      getNinjaEvents(event.id).then((response) => {
+        setNinjas((ninjas) => [...ninjas, ...response.data]);
+        setNinjas((ninjas) => {
+          return ninjas.filter(
+            (ninja, index, self) =>
+              index === self.findIndex((t) => t.id === ninja.id)
+          );
+        });
+      });
+    });
+  }, [events]);
+
+  const onFinish = (values) => {
+    if (id) {
+      api
+        .createLecture(id, values)
+        .then(() => {
+          router.push("/admin/lectures");
+        })
+        .catch((error) => notification["error"](error.data?.errors));
+    } else {
+      api
+        .createLecture(values)
+        .then((response) => {
+          router.push("/admin/lectures");
+        })
+        .catch((error) => notification["error"](error.data?.errors));
+    }
+  };
+
+  return (
+    <>
+      <Row justify="space-between">
+        <Title level={2}>
+          {id && ninjas[0]
+            ? ninjas[0].first_name + " " + ninjas[0].last_name
+            : "Nova Sessão"}
+        </Title>
+        <Space>
+          <Link href="/admin/lectures">
+            <Button
+              danger
+              shape="circle"
+              size="large"
+              icon={<CloseOutlined />}
+            />
+          </Link>
+          <Button
+            shape="circle"
+            type="primary"
+            size="large"
+            icon={<SaveOutlined />}
+            onClick={() => form.submit()}
+          />
+        </Space>
+      </Row>
+      <Row align="middle">
+        <Col xs={24} sm={24} md={20} lg={16} xl={12}>
+          <Form
+            {...{
+              labelCol: { span: 8 },
+              wrapperCol: { span: 16 },
+            }}
+            form={form}
+            onFinish={onFinish}
+          >
+            <Form.Item
+              label="Evento"
+              name="lecture[event_id]"
+              rules={[{ required: true }]}
+            >
+              <Select
+                placeholder="Escolha um evento"
+                style={{ width: "50%" }}
```

Comment:
```suggestion
                style={{ width: "50%" }}
```

Can you put a higher width? Same for the other items with such width.